Microsoft.Toolkit.Uwp/Microsoft.Toolkit.Uwp.csproj

```diff
@@ -1,11 +1,13 @@
 <Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
-    <TargetFrameworks>uap10.0.17763</TargetFrameworks>
-    <Title>Windows Community Toolkit</Title>
+    <TargetFramework>uap10.0.17763</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Title>Windows Community Toolkit - Basics</Title>
```

Comment:
@michael-hawker are you aware of this change?

Reply:
@michael-hawker

Reply:
```suggestion
    <Title>Windows Community Toolkit - UWP</Title>
```

Thanks Alex, yeah we couldn't use the term 'Basic' here. Let's just label it with the technology for now.